core/src/main/java/com/nisovin/magicspells/spells/targeted/HomingMissileSpell.java

```diff
@@ -202,6 +216,7 @@ public CastResult castAtEntityFromLocation(SpellData data) {
 	}
 
 	private class MissileTracker implements Runnable {
+		Random rand = ThreadLocalRandom.current();
```

Comment:
`Spell` class has a `random` variable, use that instead of declaring a new thread local random.